pkg/operator/staticpod/startupmonitor/cmd.go

```diff
@@ -95,6 +100,12 @@ func NewCommand(check ReadinessChecker) *cobra.Command {
 				if c, ok := o.Check.(WantsRestConfig); ok {
 					c.SetRestConfig(restConfig)
 				}
+
+				operatorClient, err := newOperatorClient(restConfig)
```

Comment:
maybe at this point, we should make `o.KubeConfig` required?

Reply:
done